### ISSUE TITLE
cmocka - the successor of cmockery

### ISSUE THREAD

**Issue by gladiac** [google/cmockery#33]
Hello,

as cmockey is unmaintained I've forked the project some years ago and created a successor called cmocka. You can find the project website here:

https://cmocka.org/

It would be great if you could point to cmocka which fixes a lot of bugs and has much more features than cmockery in the meantime.

Thanks,

 Andreas

**Comment by stewartmiles** [comment]
It would be great if you proposed merges from your fork back into the
mainline TBH.  I vaguely remember a few years ago getting some pull
requests to improve the library which required quite a bit of clean up
before we were happy about merging the patches.  If you're willing to go
through some review we would be happy to cross the streams again.

Cheers,
Stewart

**Comment by gladiac** [comment]
I'm sorry but a few years ago, I created a git tree with all the patches from your bugtracker applied and with cmake as the build system. You ignored that pull request for more than a year, so I forked the project. Now cmocka is ~300 patches ahead of your tree and we added a lot of new features. You had the chance that I merge back patches 5 years ago. I'm sorry but it is too late now. However maybe you want to give cmocka a try ...
